src/main/webapp/assets/modules/common/securityService.js

```diff
@@ -64,19 +99,62 @@ techlooper.factory("securityService", function (apiService, $rootScope, $q, util
 
           $rootScope.userInfo = data;
 
-          var lastFoot = localStorageService.get("lastFoot");
-          if (lastFoot && ["/login", "/user-type"].indexOf(lastFoot) == -1) {
-            localStorageService.remove("lastFoot");
-            return $location.path(lastFoot);
-          }
-          localStorageService.remove("lastFoot");
+          //var lastFoot = localStorageService.get("lastFoot");
+          //if (lastFoot && ["/login", "/user-type"].indexOf(lastFoot) == -1) {
+          //  localStorageService.remove("lastFoot");
+          //  return $location.path(lastFoot);
+          //}
+          //localStorageService.remove("lastFoot");
 
-          instance.routeByRole();
+          //instance.routeByRole();
         })
         .error(function () {utils.sendNotification(jsonValue.notifications.loaded, $(window).height());});
     },
 
-    init: function () {}
+    removeProtectedLastFoot: function () {
+      var path = localStorageService.get("protectedPage");
+      if (/\/freelancer\/post-project/.test(path) || /\/employer-dashboard/.test(path) || /\/post-challenge/.test(path)) {
+        localStorageService.remove("protectedPage");
+      }
+    },
+
+    initialize: function () {
+      $rootScope.$on("$routeChangeStart", function (event, next, current) {
+        var view = utils.getView();
+        switch (view) {
+          case jsonValue.views.freelancerPostProject:
+            var lastPage = "/freelancer/post-project";
+          case jsonValue.views.employerDashboard:
+            var lastPage = "/employer-dashboard";
```

Comment:
'lastPage' is already defined.<br>Expected a 'break' statement before 'case'.<br>'lastPage' is defined but never used.